apps/liquidity-provision-dashboard/src/app/components/status/status.tsx

```diff
@@ -52,14 +53,16 @@ export const Status = ({
   );
 };
 
-const getTooltipDescription = (status: string) => {
+const getTooltipDescription = (status: string, trigger?: string) => {
   let tooltipDescription = '';
   switch (status) {
     case Schema.MarketTradingModeMapping.TRADING_MODE_CONTINUOUS:
       tooltipDescription =
         'This is the standard trading mode where trades are executed whenever orders are received';
       break;
-    case `${Schema.MarketTradingModeMapping.TRADING_MODE_MONITORING_AUCTION} - ${Schema.AuctionTriggerMapping.AUCTION_TRIGGER_LIQUIDITY}`:
+    case `${
```

Comment:
suggestion (non blocking): Not to be fixed in this PR here, but I don't love having this concatenated string as a dynamic case in the switch statement. 

I think this could be improved by a small refactor to make `getStatus` and `getTooltipDescription` accept only the raw enum values (NOT the mappings) and only returning raw strings for rendering, and also not depend on each other.

Additionally I would 'mark' each returned string in getTooltipDescription by using the `t` function individually for each result.

Non urgent but ping @ciaran-

Reply:
@ciaran-  I updated these methods as described above

Reply:
That's better, yeah - thanks @MadalinaRaicu ! 

How do we feel about this once the LP dashboard moves to the website - do we still think we'd want a specific-to-the-dashboard function to do this mapping or do we think we could re-use the console component for that purpose?